snap/snapcraft.yaml

```diff
@@ -43,6 +43,7 @@ apps:
     command: etc/spark8t/launcher.sh $SNAP/lib/python3.10/site-packages/spark8t/cli/spark_submit.py
     environment:
       PYTHONPATH: $SNAP/python:$SNAP/lib/python3.10/site-packages:$PYTHONPATH
+      AWS_JAVA_V1_DISABLE_DEPRECATION_ANNOUNCEMENT: "true"
```

Comment:
Does it work if you put in the general `environment` section ?

Reply:
You're right, I've moved this to the general environment section.